exec/java-exec/src/test/java/org/apache/drill/exec/expr/fn/FunctionInitializerTest.java

```diff
@@ -45,25 +45,29 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
-import static org.mockito.Matchers.any;
-import static org.mockito.Mockito.doAnswer;
-import static org.mockito.Mockito.spy;
 
-@RunWith(MockitoJUnitRunner.class)
 @Category(SqlFunctionTest.class)
 public class FunctionInitializerTest {
 
-  private static final String CLASS_NAME = "com.drill.udf.CustomLowerFunction";
+  @ClassRule
+  public static final TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+  private static final String CLASS_NAME = "org.apache.drill.udf.dynamic.CustomLowerFunction";
   private static URLClassLoader classLoader;
 
   @BeforeClass
   public static void init() throws Exception {
-    Path jars = TestTools.WORKING_PATH
-      .resolve(TestTools.TEST_RESOURCES)
-      .resolve("jars");
-    String binaryName = "DrillUDF-1.0.jar";
-    String sourceName = JarUtil.getSourceName(binaryName);
-    URL[] urls = {jars.resolve(binaryName).toUri().toURL(), jars.resolve(sourceName).toUri().toURL()};
+    File buildDirectory = temporaryFolder.getRoot();
+    String binaryName = "drill-custom-lower";
+
+    JarBuilder jarBuilder = new JarBuilder("src/test/resources/drill-udf");
+    int result = jarBuilder.build(binaryName, buildDirectory.getAbsolutePath(), "**/CustomLowerFunction.java", null);
+    assertEquals("Build should be successful.", 0, result);
```

Comment:
Consider moving `assertEquals` inside `JarBuilder.build()` and returning name of the binary jar instead.